README.md

```diff
@@ -6,6 +6,14 @@ sentry internal pypi
 this repository contains the tools to import and/or build packages from public pypi for the
 platforms and achitectures required for sentry development.
 
+## setup
+
+```
+python3 -m venv .venv
+source .venv
+pip install packaging wheel
+```
+
```

Comment:
the expectation for tox-based packages is to do `tox --devenv venv` (or `tox d venv` for tox 4.x) -- and then you both (1) don't need directions in each repo since tox is standardized and (2) don't need gitignore because `virtualenv` auto-ignores it

Reply:
@asottile-sentry thanks, I was able to set up the venv:

```
pypi on  add-setup-instructions via 🐍 v3.10.6 on ☁️  michael@sentry.io(us-central1) took 2s
❯ tox --version
4.2.6 from /opt/homebrew/Cellar/tox/4.2.6/libexec/lib/python3.11/site-packages/tox/__init__.py

pypi on  add-setup-instructions via 🐍 v3.10.6 on ☁️  michael@sentry.io(us-central1)
❯ tox d venv
py: install_deps> python -I -m pip install covdefaults coverage pytest re-assert -r docker/requirements.txt
  py: OK (11.28 seconds)
  congratulations :) (11.31 seconds)
ROOT: created development environment under /Users/mwarkentin/repos/sentry/pypi/venv
```

Once that's in place is this the expected way to run within the devenv venv?

```
tox exec -- python3 -m add_pkg google-cloud-secret-manager
```

Reply:
If that looks good I can update this PR to document those steps as it wasn't entirely clear just from visiting the repo that everything was expected to be done from within `tox`.

Reply:
nah you'd just use the virtualenv like a normal virtualenv -- `. venv/bin/activate` and then normal stuff

Reply:
Updated.